lms/templates/learner_dashboard/program_header_view.underscore

```diff
@@ -1,24 +1,39 @@
-<picture>
-    <source srcset="<%- banner_image_urls.w1440h480 %>" media="(min-width: <%- breakpoints.min['x-large'] %>)">
-    <source srcset="<%- banner_image_urls.w726h242 %>" media="(min-width: <%- breakpoints.min.medium %>)">
-    <img class="banner-image" srcset="<%- banner_image_urls.w348h116 %>" alt="">
-</picture>
-<h2 class="hd-2 title"><%- name %></h2>
-<p class="subtitle"><%- subtitle %></p>
-<a href="<%- program_details_url %>" class="breadcrumb"><%- gettext('Programs') %></a>
-<span><%- StringUtils.interpolate(
-    gettext('{category}\'s program'),
-    {category: category}
-) %></span>
-<% _.each(organizations, function(org) { %>
-	<img src="<%- org.img %>" class="org-logo" alt="<%- StringUtils.interpolate(
-	    gettext('{organization}\'s logo'),
-	    {organization: org.display_name}
-	) %>">
-<% }) %>
-<% if (category === 'xseries') { %>
-	<p><%- StringUtils.interpolate(
-        gettext('To complete the {program} XSeries and earn an XSeries Certificate you must successfully earn a Verified Certificate in all courses shown below.'),
-        {program: name}
-    ) %></p>
-<% } %>
+<div class="banner-background-wrapper">
+    <picture>
+        <source srcset="<%- banner_image_urls.w1440h480 %>" media="(min-width: <%- breakpoints.min['x-large'] %>)">
+        <source srcset="<%- banner_image_urls.w726h242 %>" media="(min-width: <%- breakpoints.min.medium %>)">
+        <img class="banner-background-image" srcset="<%- banner_image_urls.w348h116 %>" alt="">
+    </picture>
+</div>
+    
+<div class="banner-content grid-container">
+    <h2 class="hd-1 title"><%- name %></h2>
+    <p class="hd-4 subtitle"><%- subtitle %></p>
+    <% if (organizations.length) { %>
+        <div class="org-wrapper">
+          <% _.each(organizations, function(org) { %>
+            <img src="<%- org.img %>" class="org-logo" alt="<%- StringUtils.interpolate(
+                gettext('{organization}\'s logo'),
+                {organization: org.display_name}
+            ) %>">
+          <% }) %>
+        </div>
+    <% } %>
+</div>
+
+<nav class="breadcrumb-wrapper grid-container" role="navigation">
```

Comment:
Why do we need the `grid-container` class name here if we are not using `col` classes in the contained elements?
 Add a line note